kext/FileNVRAM/FileNVRAM.h

```diff
@@ -180,7 +182,7 @@ class FileNVRAM : public IODTNVRAM
     OSDictionary * mNvramMissDict;
     IOCommandGate* mCommandGate;
     OSString*      mFilePath;
-    IOTimerEventSource* mTimer;
+    IONotifier* disk_notifier_start_;
```

Comment:
I've just deleted some commits to have a more clear history. I would leave the changes for the FileNVRAM module there for now (my own purpose). Sir you don't need this pull request for that little change and we can close it, just apply by your self. Later, after testing with chameleon I will open a new pull request.... I have to test the internal kernel/kexts patcher using the module and ensure that no conflicts exists or make it conditional when the module is loaded. Thanks for the patience!